### ISSUE TITLE
Formatting messes up code when not using self-closing tags (HTML5) for <br> or <img>

### ISSUE THREAD

**Issue by ignism** [mblode/vscode-twig-language-2#31]
When I format HTML5 type markup, the formatter messes up my code. Because it is not able to handle non closed tags like `<br>`, `<img>` or `<meta>`

For instance this simple page:

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
</head>
<body>
    
</body>
</html>
```

Turns into this abomination, after formatting:

```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
            <meta content="width=device-width, initial-scale=1.0" name="viewport">
                <meta content="ie=edge" http-equiv="X-UA-Compatible">
                    <title>Document</title>
                </head>
                <body></body>
            </html>
```

**Comment by ImKubass** [comment]
Downgrading to 0.3.1 version works for me

**Comment by mblode** [comment]
@ignism I'll get into contact with the developer behind Pretty Diff about this issue. Thanks 👍

**Comment by mblode** [comment]
Ok, I have pushed an update where the formatter will interpret the code as HTML instead of Twig and it should fix this closed tag issue. Previously setting the language as TWIG, it thought the code was XML which is very strict about having closed tags but HTML is laxer 😄

**Comment by mblode** [comment]
Fixed HTML void-tag handling in 0.12.0 and added regression tests.
